ncm-network/src/main/perl/network.pm

```diff
@@ -1562,9 +1590,14 @@ sub start
     } elsif (@ifaces) {
         my @cmds;
         foreach my $iface (@ifaces) {
-            push(@cmds, ["/sbin/ifup", $iface, "boot"]);
+            if ($self->_is_executable($IFUP_CMD)) {
```

Comment:
this requires the networkmanager compatibility rpm?

Reply:
ok, not requires, but does it work when that rpm is present?

Reply:
yes, it will use the legacy ifup/ifdown scripts if they are installed, otherwise it will use nmcli (if allowed)

Reply:
tbh i would require the ifup/ifdown being present, and not implement your own version of it like the link removal code if ifdown is not present.

Reply:
OK, I'll revert this.